plugins/services/src/js/components/modals/CreateServiceModalForm.js

```diff
@@ -628,10 +652,16 @@ class CreateServiceModalForm extends Component {
             onRemoveItem={this.handleRemoveItem}
             onAddItem={this.handleAddItem}
           />
-        </TabView>
+        </TabView>,
+        ...pluginTabViews
```

Comment:
Could we leverage `<Mount />` component here? And then just mount additional tabs like so https://github.com/mesosphere/dcos-ui-plugins-private/blob/master/secrets/hooks.js#L64 ?

Reply:
Given how TabViewList works I don’t think Mount will work here. I was going to try that initially but opted for this instead.